src/lib/modules/Document/utils.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Truncate } from '@components/Truncate';
+import find from 'lodash/find';
```

Comment:
Super minor: we normally add a `low dash` :) as prefix to immediately remind that this is a `lodash` func:
```suggestion
import _find from 'lodash/find';
```